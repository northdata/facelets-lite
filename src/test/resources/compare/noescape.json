{
	"text": "<p>One</p><p>Two</p>"
}