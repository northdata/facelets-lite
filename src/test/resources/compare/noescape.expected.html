<!DOCTYPE html>
<html xmlns:h="http://java.sun.com/jsf/html">
	<h1>Headline</h1>
	<p>One</p>
	<p>Two</p>
</html>